test/verify/check-networking

```diff
@@ -420,5 +420,33 @@ NM_CONTROLLED=no
         self.browser.wait_present(sel)
         self.browser.wait_visible(sel)
 
+    @unittest.skipIf("fedora-25" not in os.environ.get("TEST_OS", ""), "No network checkpoints here.")
```

Comment:
Checkpoints have been added to NetworkManager 1.4, which is only available in Fedora 25.